ReactiveCocoa/AppKit/NSButton.swift

```diff
@@ -4,8 +4,24 @@ import AppKit
 
 extension Reactive where Base: NSButton {
 
-	public var pressed: Signal<Base, NoError> {
-		return trigger.map { [unowned base = self.base] in base }
+	internal var associatedAction: Atomic<(action: CocoaAction<Base>, disposable: Disposable?)?> {
+		return associatedValue { _ in Atomic(nil) }
 	}
 
+	public var pressed: CocoaAction<Base>? {
+		get {
+			return associatedAction.value?.action
+		}
+
+		nonmutating set {
+			base.target = newValue
```

Comment:
Can you use `trigger` instead of assigning `CocoaAction` directly? Otherwise, this would override `trigger` when used together.

I'd expect the caveat to be solved soon, but for now let's reduce the reach of it.